src/main.rs

```diff
@@ -48,47 +50,59 @@ fn main() -> Result<(), Box<dyn Error>> {
         }
     } else if let Some(input) = cli.input {
         let input = String::from(input.to_string_lossy());
+        let code = {
+            if input == "-" || input == "--silence" {
```

Comment:
It was correct before:

```suggestion
            if input == "-" {
```